package.json

```diff
@@ -13,24 +13,24 @@
 	"unpkg": "dist/browser.umd.js",
 	"exports": {
 		"./browser": {
+			"types": "./dist/browser.d.ts",
 			"import": "./dist/browser.mjs",
-			"require": "./dist/browser.js",
-			"types": "./dist/browser.d.ts"
+			"require": "./dist/browser.js"
 		},
 		"./cairo": {
+			"types": "./dist/cairo.d.ts",
 			"import": "./dist/cairo.mjs",
-			"require": "./dist/cairo.js",
-			"types": "./dist/cairo.d.ts"
+			"require": "./dist/cairo.js"
 		},
 		"./napi-rs": {
+			"types": "./dist/napi-rs.d.ts",
 			"import": "./dist/napi-rs.mjs",
-			"require": "./dist/napi-rs.js",
-			"types": "./dist/napi-rs.d.ts"
+			"require": "./dist/napi-rs.js"
 		},
 		"./skia": {
+			"types": "./dist/skia.d.ts",
 			"import": "./dist/skia.mjs",
-			"require": "./dist/skia.js",
-			"types": "./dist/skia.d.ts"
+			"require": "./dist/skia.js"
```

Comment:
shouldnt these refer to the root files then

Reply:
No, that would make Node.js load extra code and go thru one more import hop.

The export files are merely for compatibility with older versions of Node.js, as well as TypeScript and also old versions of other tools, such as Jest.